docs/examples/disablingsignalhandlerinstallation.rst

```diff
@@ -0,0 +1,22 @@
+================================================
+Example -- Disabling signal handler installation
+================================================
+
+You have the possibility to print raised errors directly in the file descriptor specified in the ``Klein.run()`` ``logFile`` parameter, it can be useful for troubleshooting or logging fins.
```

Comment:
There are reasons to want to have this parameter exposed, but … this is not one of them. Twisted logs raised errors automatically already, so this documentation just seems inaccurate?  You also don't actually *use* the `logFile` parameter here, despite describing it.  So this seems like an odd corner of the documentation.

It might be better to document how to call `reactor.run` or integrate Klein with `twisted.internet.task.react` directly, rather than copying every possible parameter into the `run` convenience function.